packages/plugins/plugin-sheet/src/components/GridSheet/util.ts

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { useEffect, useState } from 'react';
+
+import { createDocAccessor } from '@dxos/react-client/echo';
+import { type GridEditing, type GridContentProps } from '@dxos/react-ui-grid';
+import { mx } from '@dxos/react-ui-theme';
+
+import { addressFromIndex, type CellAddress, type SheetModel, type FormattingModel } from '../../model';
+
+export const dxGridCellIndexToSheetCellAddress = (gridIndex: GridEditing): CellAddress | null => {
+  if (!gridIndex) {
+    return null;
+  }
+  const [colStr, rowStr] = gridIndex.split(',');
+  return {
+    column: parseInt(colStr),
+    row: parseInt(rowStr),
+  };
+};
+
+const renderDxGridCells = (model: SheetModel, formatting: FormattingModel) => {
+  return Object.keys(model.sheet.cells).reduce((acc: NonNullable<GridContentProps['cells']>, sheetCellIndex) => {
+    const address = addressFromIndex(model.sheet, sheetCellIndex);
+    const cell = formatting.getFormatting(address);
+    if (cell.value) {
+      acc[`${address.column},${address.row}`] = { value: cell.value, className: mx(cell.classNames) };
```

Comment:
```
= {
  value: cell.value,
  className: ...,
}
```

Reply:
per our chat if this handles the computed value i might put a comment here.

Reply:
I’m not sure what you mean… is the variable name `cell` misleading perhaps? The code block you pasted looks equivalent to what’s here.